astroquery/mast/missions.py

```diff
@@ -28,7 +30,7 @@ class MastMissionsClass(MastQueryWithLogin):
     """
     MastMissions search class.
 
-    Class that allows direct programatic access to the MAST search API for a given mission.
+    Class that allows direct programatic access to retrieve metadata via the MAST search API for a given mission.
     """
 
     def __init__(self, *, mission='hst', service='search'):
```

Comment:
Hi @syed-gilani this PR is looking good, one quick request: can you make the optional arguments keyword arguments only like how you did here?